netbout-client/pom.xml

```diff
@@ -178,6 +178,7 @@
                     <plugin>
                         <groupId>com.qulice</groupId>
                         <artifactId>qulice-maven-plugin</artifactId>
+                        <version>0.15.2</version>
```

Comment:
@danff you should remove this and add pluginManagement section in netbout/pom.xml with this qulice version